src/proxy.rs

```diff
@@ -152,7 +170,15 @@ async fn metrics(proxy: Arc<Proxy>) -> Result<Response<Body>, GenericError> {
 
     let output = String::from_utf8(buffer.clone());
     buffer.clear();
-    Ok(Response::new(Body::from(output.unwrap_or_default())))
+    Ok(Response::builder()
+        .header(hyper::header::ACCESS_CONTROL_ALLOW_ORIGIN, "*")
```

Comment:
The value should be in config, similar comments for all other instances of adding the CORS header